seqr-gke-nodepool/variables.tf

```diff
@@ -39,3 +35,15 @@ variable "node_pool_local_ssd_count" {
   type    = string
   default = "0"
 }
+
+variable "min_node_pool_count" {
```

Comment:
I opted to override `min_node_pool_count` with dual meanings here rather than keep a third `node_pool_count` variable around.